samples/boards/esp32/flash_encryption/boards/esp32.overlay

```diff
@@ -7,3 +7,9 @@
 &flash0 {
```

Comment:
typo in this commit: `samples/boards/esp32: Add chosen zpehyr,code-partition to overlay` *zephyr